requirements_examples.txt

```diff
@@ -9,4 +9,5 @@ matplotlib
 cffi
 gmplot
 pycurl
+types-pycurl
```

Comment:
this goes in the pre-commit config. see the openpilot pre-commit config for an example

Reply:
Thanks